### PR TITLE
FRW-19 Support Symfony packages v6.

### PR DESCRIPTION
**Pull request by geega**
- Developer(s): @geega 

- Ticket: https://spryker.atlassian.net/browse/FRW-19

- Release Group: https://release.spryker.com/release-groups/view/4549

- merge: merge

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   ArchitectureSniffer   | patch (currently 0.x)  |                       |

-----------------------------------------

### Improvements

- Adjusted `composer.json` to allow for new `symfony/finder:^6.0.0` major.

### CODE REVIEW

**Review by dimitriyTsemma — APPROVED** [review]

**Review by mstfEmkSpryker — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         php-version: [
             '7.4',
-            '8.1'
+            '8.2'
```
Comment (asmarovydlo) [inline .github/workflows/ci.yml:18]: How it's related to symfony update?

**Review by asmarovydlo — CHANGES_REQUESTED** [review]
Please add changelogs, release table.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (geega) [reply .github/workflows/ci.yml:18]: @asmarovydlo this changes make by @dereuromark. Looks like for check validation 
https://github.com/spryker/architecture-sniffer/pull/117/commits/c2e4275f2d4da4ea76a0d9e3ab67c838816b73ef

Comment (asmarovydlo) [reply .github/workflows/ci.yml:18]: @dereuromark we have a separate ticket for updating to 8.2, do we really want to make it here?

Comment (dereuromark) [reply .github/workflows/ci.yml:18]: Both is fine

Comment (dereuromark) [reply .github/workflows/ci.yml:18]: As CI already teils u

**Review by asmarovydlo — CHANGES_REQUESTED** [review]
Removed `ready for release` label, because this package is used on `suite` in a specific version, and without a tag in won't be available on packagist, so we can't make it work there together with the latest Symfony packages.

So, https://github.com/spryker/architecture-sniffer/pull/117#pullrequestreview-1149699642 is still actually, please fix it.

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**